client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -188,8 +196,38 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 			renderValidationError( trademarkNumberStrings[ error ] )
 		);
 
+		// Note organization is the level above the other extra fields
+		const organizationValidationStrings = {
+			maxLength: translate( 'Too long, please limit the organization name to 100 characters.' ),
+			not: translate( 'Please use only the characters “{{validCharacters}}”',
+				{ args: { validCharacters: 'a-z A-Z 0-9 . , ( ) @ & \' - [space]' } }
+			),
+			$ref: translate( 'Organization field is required' ),
+		};
+
+		const organizationValidationMessage = map( contactDetailsValidationErrors.organization, error =>
+			renderValidationError( organizationValidationStrings[ error ] )
+		);
+
 		return (
 			<div>
+				<FormFieldset>
+					<FormLabel className="registrant-extra-info__organization" htmlFor="organization">
+						{ translate( 'Organization Name' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **15** times:
`translate( 'Add Organization Name' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/cctlds/add-organization-to-tld-forms)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).